### PR TITLE
[4.0] mod_quickicons

### PR DESCRIPTION
**Pull request by brianteeman**
We really should be using the term "non-core" and not "3rd party"

To test install any extension that adds its own quickicon - eg akeeba

image

(The missing image is unrelated to this pr)

### CODE REVIEW

**Comment by N6REJ** [comment]
Why do you say that brian?  Very few designers & end users have any idea what a "core" is but all understand "3rd party"

**Comment by brianteeman** [comment]
3rd party is derogatory

**Comment by HLeithner** [comment]
Even if I don't think it's derogatory. Non-core sounds better for me.

File: administrator/components/com_admin/sql/updates/postgresql/4.0.0-2019-09-15.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+UPDATE "#__modules" SET "title" = 'Non-core Extensions' WHERE "#__modules"."asset_id" = 82;
+UPDATE "#__assets" SET "title" = 'Non-core Extensions' WHERE "#__assets"."asset_id" = 82;
+
```
Comment (Quy) [inline administrator/components/com_admin/sql/updates/postgresql/4.0.0-2019-09-15.sql]:
```suggestion
```

**Comment by N6REJ** [comment]
How is it derogatory?  It's an industry standard term for products the manufacturer doesn't make that work with their product(s).
https://en.wikipedia.org/wiki/Third-party_source

**Comment by ot2sen** [comment]
@N6REJ Joomla! is different in the industry by being 100% community run. This means that extension developers, template creators and so on all are part of the whole. So they are at the party making it great :)

File: administrator/components/com_admin/sql/updates/postgresql/4.0.0-2019-09-15.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+UPDATE "#__modules" SET "title" = 'Non-core Extensions' WHERE "#__modules"."asset_id" = 82;
+UPDATE "#__assets" SET "title" = 'Non-core Extensions' WHERE "#__assets"."asset_id" = 82;
```
Comment (Quy) [inline administrator/components/com_admin/sql/updates/postgresql/4.0.0-2019-09-15.sql]:
```suggestion
UPDATE "#__assets" SET "title" = 'Non-core Extensions' WHERE "#__assets"."id" = 82;
```

**Comment by simbus82** [comment]
"Non-core" is really unreadable and unlistenable... it is also a term related to "business" and "assets management", rarely used in information technology.

"3rd-party" is the right and standard definition.
Maybe we can use the term "Add-on Extensions" or "Additional Extensions"

**Comment by brianteeman** [comment]
I will just leave this here

image

**Comment by infograf768** [comment]
I don’t care for the module name as it can be modified, but it is clear we will not use it for the filter in French as « Extensions ne faisant pas partie du noyau » would be ridiculously long. « Extensions tierces » not only is not derogatory at all but corresponds exactly to what these are.

**Comment by brianteeman** [comment]
3rd party is also not accurate as extensions by joomla would be inn the same list

**Comment by infograf768** [comment]
Any extension which is taken care of, debugged, etc. by the project itself on this github should be considered as `core`.
Until https://github.com/joomla/joomla-cms/pull/26329 was merged webservices were not considered as core.

**Comment by brianteeman** [comment]
patchtester, weblinks, all language packs are all non-core - ie they do not ship with the core of joomla. but they are definitely not 3rd party. Do what you want in your translations but on english it is correct

**Comment by infograf768** [comment]
Language packs are 3rd party, whatever you decide unilaterally.
Anything which is not present in `libraries/src/Extension/ExtensionHelper.php` is 3rd party as it is filtered this way in Extensions=> Manage.
And yes, we certainly will do what we think is right for our languages. 
It seems though that I am not the only one to consider the expression `Non-core` as incorrect.
@simbus82 is totally right.

**Comment by N6REJ** [comment]
> Do what you want in your translations but on english it is correct

No, in your MIND it's correct.  For 14yrs its been the defacto verbage and now suddenly it's derogatory?
Are we building Joomla for developer/coders?  or are we building for real people?
As coders our brains don't work the way everyone else's does.  We see patterns, processes issues, just as easily as breathing.
This is NOT the way the majority of people are taught to think.

**Comment by nadjak77** [comment]
I have tested this item :white_check_mark: successfully on b1982e83fb49bf93e41d394b570d20953c023c22<br /><br />title changed from "3rd Party" to "non-core Extensions"<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26311">issues.joomla.org/tracker/joomla-cms/26311</a>.</sub>

**Comment by neotrendch** [comment]
I have tested this item :white_check_mark: successfully on b1982e83fb49bf93e41d394b570d20953c023c22<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26311">issues.joomla.org/tracker/joomla-cms/26311</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26311">issues.joomla.org/tracker/joomla-cms/26311</a>.</sub>

**Comment by pe7er** [comment]
I have tested this item :red_circle: unsuccessfully on b1982e83fb49bf93e41d394b570d20953c023c22<br /><br />We have tested this patch on https://start.joomlapbf.com/joomla42/ 
The Quick Icon on the dashboard does not change.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26311">issues.joomla.org/tracker/joomla-cms/26311</a>.</sub>

## After applying patch
We have installed Akeeba Backup via System > Install Extensions > [Upload Extension] 
and the Dashboard looks like this: 
pbf-Screenshot_2019-10-19_11-36-03

**Comment by rmittl** [comment]
This filter option in Extension is ok, but the Quick Icon in Dashboard has not changed.

Tested unsuccessful.

**Comment by karo3** [comment]
I have tested this item :red_circle: unsuccessfully on b1982e83fb49bf93e41d394b570d20953c023c22<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26311">issues.joomla.org/tracker/joomla-cms/26311</a>.</sub>

**Comment by webfeuerflo** [comment]
I have tested this item :red_circle: unsuccessfully on b1982e83fb49bf93e41d394b570d20953c023c22<br /><br />Quick Item Title is still "3rd Party".

screenshot<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26311">issues.joomla.org/tracker/joomla-cms/26311</a>.</sub>

**Comment by webfeuerflo** [comment]
I have tested this item :white_check_mark: successfully on b1982e83fb49bf93e41d394b570d20953c023c22<br /><br />After executing the SQL query in PHPmyadmin the test was successful<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26311">issues.joomla.org/tracker/joomla-cms/26311</a>.</sub>

**Comment by N6REJ** [comment]
@simbus82
> Maybe we can use the term "Add-on Extensions" or "Additional Extensions"

exactly!

**Comment by brianteeman** [comment]
Might as well say extension extensions

**Comment by brianteeman** [comment]
cant be bothered - people seem to be happy with inconsistency

**Pull request closed without merging**